.github/workflows/daily-nightly-jobs.yml

```diff
@@ -1,7 +1,10 @@
 name: Daily nightly jobs
 on:
-  schedule:
-    - cron: "0 0 * * *" # every day at midnight
+  pull_request:
```

Comment:
This should run for midnight job, I hope you will revert this

Reply:
yes, those were just temporary changes for testing